drivers/can/can_mcux_mcan.c

```diff
@@ -170,13 +190,26 @@ static const struct can_driver_api mcux_mcan_driver_api = {
 #endif /* CONFIG_CAN_FD_MODE */
 };
 
+struct can_mcan_ops mcux_mcan_ops = {
```

Comment:
Could this be const?
(same in other drivers)

Reply:
Yes! That was the intention too, fixed. Thanks.

Reply:
Could you add `static` too

Reply:
Just did, thanks.